.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
+yarnPath: .yarn/releases/yarn-3.1.1.cjs
```

Comment:
What is this file for? Are we supposed to maintain a copy of yarn 3.1.1 in the repo?